integration-tests/springmvc-tests/pom.xml

```diff
@@ -37,6 +37,8 @@
     <module>springmvc-tests-general-with-springboot</module>
     <module>springmvc-tests-simplified-mapping-with-springboot</module>
     <module>springmvc-tests-connection-limit</module>
+    <module>springmvc-tests-general-with-springboot2</module>
```

Comment:
new integration tests:
  - IMPL in it-producer, test from it-consumer
  - spring boot2 is just a deploy mode